src/foundry/client/apps/forms/journal-page-sheet.d.ts

```diff
@@ -0,0 +1,292 @@
+import type { EditorView } from "prosemirror-view";
+import type Showdown from "showdown";
+import type { Editor } from "tinymce";
+import type { JOURNAL_ENTRY_PAGE_FORMATS } from "../../../common/constants.mjs.js";
+import type { CONST } from "../../../common/module.mjs.js";
+
+declare global {
+  /**
+   * The Application responsible for displaying and editing a single {@link JournalEntryPage} document.
+   *
+   * @typeParam TOptions - The type of the options object
+   */
+  class JournalPageSheet<TOptions extends JournalPageSheet.Options = JournalPageSheet.Options> extends DocumentSheet<
+    TOptions,
+    JournalEntryPage
+  > {
+    /**
+     * @param object - The JournalEntryPage instance which is being edited.
+     * @param options - Application options
```

Comment:
nit; the `-`s in `@param`s should be aligned:
```suggestion
     * @param object  - The JournalEntryPage instance which is being edited.
     * @param options - Application options
```
I think this'd do it though maybe it looks aligned in editor?